packages/grafana-data/src/types/transformations.ts

```diff
@@ -2,8 +2,17 @@ import { MonoTypeOperatorFunction } from 'rxjs';
 
 import { RegistryItemWithOptions } from '../utils/Registry';
 
-import { ScopedVars } from './ScopedVars';
 import { DataFrame, Field } from './dataFrame';
+import { InterpolateFunction } from './panel';
+import { DataQuery } from './query';
+
+/**
+ * Context passed to transformDataFrame and to each transform operator
+ */
+export interface DataTransformContext {
+  interpolate: InterpolateFunction;
+  queries?: DataQuery[];
```

Comment:
^^^ should the queries be removed/commented out since it is not yet used or implemented?

Reply:
yes